media_avportal.post_update.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Post updates functions for Media AV Portal.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Sets caching of AV Portal responses to one hour.
+ */
+function media_avportal_post_update_00001(): void {
```

Comment:
Should we force to use our approach on naming of post_update. In fact AV Portal is contrib module.

Reply:
Uhm good point... No idea. We need to ask @ademarco .

Reply:
Agreed, we need to change this.